src/test/ui/impl-trait/issue-57200.rs

```diff
@@ -0,0 +1,14 @@
+// Regression test for #57200
```

Comment:
Please leave a FIXME here so that we will revisit this at some point. The error is due to a temporary hack.